lti_emailer/management/commands/report_user_primary_email.py

```diff
@@ -0,0 +1,85 @@
+import logging
+import csv
+import os
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+
+from icommons_common.models import Term, CourseInstance
+
+from canvas_sdk.utils import get_all_list_data
+from canvas_sdk.methods.courses import list_users_in_course_users
+from canvas_sdk.exceptions import CanvasAPIError
+
+from icommons_common.canvas_utils import SessionInactivityExpirationRC
+from icommons_common.models import Person
+
+
+SDK_CONTEXT = SessionInactivityExpirationRC(**settings.CANVAS_SDK_SETTINGS)
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = """
+        Creates a report of Canvas users enrolled in any course in the given term who have changed their primary email
+        address from their default SIS email address.
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument('term_id', nargs=1, type=int)
+
+    def handle(self, *args, **options):
+        term_id = options['term_id'][0]
+        try:
+            term = Term.objects.get(term_id=term_id)
+        except Term.DoesNotExist:
+            CommandError('Invalid term_id %d provided.' % term_id)
+        logger.info("Creating Canvas user primary email report for term_id %d...", term_id)
+
+        report_data = {}
+        for ci in CourseInstance.objects.filter(term=term):
+            if ci.canvas_course_id:
```

Comment:
Could you do something like:
```
ci_list = [ci in CourseInstance.objects.filter(term=term) if ci.canvas_course_id]
for ci in ci_list:
```
This would buy you precious whitespace.